### PR TITLE
Update host upload options in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/24570

@jelkosz Please review if there is sufficient information in the new steps I provided. If not, please provide any additional information that I need to add. Thanks.

### CODE REVIEW

File: multicluster_engine/add_hosts_infra_env.adoc [hunk 1]
```diff
@@ -11,9 +11,9 @@ Complete the following steps to add a host:
 
 . Select the _Hosts_ tab to view the hosts that are already added to that environment and to add a host. Available hosts might take a few minutes to appear in the table. 
 
-. Select *Discovery ISO* or *Baseboard Management Controller (BMC)* to enter the information for your host. 
+. Select *Using a Discovery ISO*, *With BMC form* or *By uploading a YAML* to enter the information for your host.
```
Comment (cadawson) [inline multicluster_engine/add_hosts_infra_env.adoc]: We generally include the Oxford comma.  Select *Using a Discovery ISO*, *With BMC form*, or *By uploading a YAML*.

File: multicluster_engine/add_hosts_infra_env.adoc [hunk 2]
```diff
@@ -33,4 +33,12 @@ Complete the following steps to add a host:
 +
 When you add a host by using the BMC option, the host is automatically approved.
 
+. If you select the *By uploading a YAML* option, complete the following steps:
++
+*Note:* The YAML option allows you to upload multiple hosts at once.
```
Comment (cadawson) [inline multicluster_engine/add_hosts_infra_env.adoc]: Suggestion: *Note:* You can upload multiple hosts at the same time when you use the YAML option.

**Review by cadawson — CHANGES_REQUESTED** [review]
Thanks Oliver. Looks good, but added a couple of comments. Looks good after those considerations.

**Comment by jelkosz** [comment]
from technical perspective lgtm

**Pull request merged**